.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,7 +5,7 @@
 
 * [ ] Testing
   - Unit test added
-  - __CLI change?:__ manually run [integration tests](https://github.com/awslabs/aws-cdk/blob/master/packages/aws-cdk/integ-tests/test.sh) and paste output as a PR comment
+  - __CLI change?:__ coordinate update of integration tests with team
```

Comment:
I'd fancy an automated run of integration tests as a PR check. Care to create an issue for that?

Ideally, CLI changes should be backwards-compatible (the CLI is an API as any other), and I worry that there is no way for a 3P contributor to have "fast" feedback on whether they broke something or not...

Reply:
We plan to move the test scripts themselves to this repo: https://github.com/awslabs/cdk-ops/issues/235

I think this should address your concerns for contributors not to be able to run those tests.

I am not 100% sure I agree that we should run all integ tests as a PR check. Ideally those should be "last resort protections". Granted, that's not true for the CLI, given we don't have good unit test coverage...